library/compiler-builtins/compiler-builtins/src/aarch64_linux.rs

```diff
@@ -137,6 +178,11 @@ macro_rules! compare_and_swap {
             ) -> int_ty!($bytes) {
                 // We can't use `AtomicI8::compare_and_swap`; we *are* compare_and_swap.
                 core::arch::naked_asm! {
+                    jmp_if_no_lse!("8f"),
```

Comment:
It would be better if this macro instead took the LSE instruction as an argument and included the `ret` and `8:`. This would reduce the code here to a single line.

Reply:
Ok, updated.